pkgs/development/python-modules/apted/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  unittestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "apted";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "JoaoFelipe";
+    repo = "apted";
+    rev = "828b3e3f4c053f7d35f0b55b0d5597e8041719ac";
+    hash = "sha256-h8vJDC5TPpyhDxm1sHiXPegPB2eorEgyrNqzQOzSge8=";
+  };
+
+  build-system = [ setuptools ];
+  nativeCheckInputs = [ unittestCheckHook ];
+
```

Comment:
Could you please add `pythonImportsCheck` for a mock test?
Besides this LGTM.

Reply:
Thanks, I've added the import check.